### DISCUSSION TITLE
I want to check the image file.

### DISCUSSION THREAD

**Discussion by younjunsu** [coder/code-server#6274] [Q&A]
After installing "code-server" and accessing Visual Studio Code through the web, I want to know how to view images. Can you provide the instructions?

# WEB VERSION
![image](https://github.com/coder/code-server/assets/27267460/05ecca6b-8976-436e-9882-5d54c58ec3bc)

# DESKTOP VERSION
![image](https://github.com/coder/code-server/assets/27267460/8f5fb663-1ef0-40e2-9fee-51ff7aab2c30)

**Comment by code-asher** [comment]
This is supposed to work out of the box.  Are there any errors in 
the browser console?
